docs/configuration.md

```diff
@@ -0,0 +1,25 @@
+# Configuration
+
+This section describes structure of configuration file.
```

Comment:
This document*

Reply:
Right, didn't manage to find correct word